### PR TITLE
ref(js): Update custom Node Otel setup docs

### PR DESCRIPTION
**Pull request by Lms24**
## DESCRIBE YOUR PR

This PR updates the Custom Node Otel setup page by:

- showing how users can now add their own HttpInstrumentation (possible since we added `httpIntegration({spans: false})` in 8.35.0
- adding a section discussing ESM loaders and that only one loader hook (either their own or Sentry's) should be registered (we learned about this in https://github.com/getsentry/sentry-javascript/issues/14065)

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [x] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/platforms/javascript/common/opentelemetry/custom-setup.mdx [hunk 1]
```diff
@@ -111,7 +111,19 @@ If you want to add your own http/node-fetch instrumentation, you have to follow
 
 ### Custom HTTP Instrumentation
```
Comment (mydea) [inline docs/platforms/javascript/common/opentelemetry/custom-setup.mdx:112]: Above there is also this sentence:

> These are needed to make sure that trace propagation works correctly. Additionally, the HTTP instrumentation is configured to ensure that request isolation is automatically applied for Sentry.

Which we should also adjust (I guess just remove the second sentence?)

File: docs/platforms/javascript/common/opentelemetry/custom-setup.mdx [hunk 2]
```diff
@@ -168,3 +180,26 @@ const provider = new NodeTracerProvider({
 // Validate that the setup is correct
 Sentry.validateOpenTelemetrySetup();
 ```
+## ESM loaders
```
Comment (mydea) [inline docs/platforms/javascript/common/opentelemetry/custom-setup.mdx:183]:
```suggestion
## ESM Loaders
```

File: docs/platforms/javascript/common/opentelemetry/custom-setup.mdx [hunk 3]
```diff
@@ -168,3 +180,26 @@ const provider = new NodeTracerProvider({
...
+
+<Note>
+
+Registering loader hooks multiple times might result in duplicated spans being created.
+[More details.](https://github.com/getsentry/sentry-javascript/issues/14065#issuecomment-2435546961)
+
+</Note>
+
+Alternatively, you can also use Sentry's loader hook and remove your own loader hook registration code.
+In this case, ensure that you initialize the Sentry SDK in its own file and you load this file prior to your application via `node --import instrument.mjs your-app.mjs`.
```
Comment (mydea) [inline docs/platforms/javascript/common/opentelemetry/custom-setup.mdx:204]:
```suggestion
In this case, ensure that you initialize the Sentry SDK in its own file and load this file prior to your application via `node --import instrument.mjs your-app.mjs`.
```

File: docs/platforms/javascript/common/opentelemetry/custom-setup.mdx (hunk 1 above)
Comment (Lms24) [reply docs/platforms/javascript/common/opentelemetry/custom-setup.mdx:112]: Right, good point. We no longer configure HttpInstrumentation for this but use our own instrumentation. Will remove the sentence

**Review by mydea — APPROVED** [review]

**Review by s1gr1d — APPROVED** [review]

**Pull request merged**